ns/rdf.ttl

```diff
@@ -167,6 +167,37 @@ rdf:direction a rdf:Property ;
 	rdfs:isDefinedBy <http://www.w3.org/1999/02/22-rdf-syntax-ns#> ;
 	rdfs:seeAlso <https://www.w3.org/TR/json-ld11/#the-rdf-compoundliteral-class-and-the-rdf-language-and-rdf-direction-properties> .
 
+rdf:TripleForm a rdfs:Class ;
```

Comment:
"triple form" isn't in RDF Concepts and a triple term _is_ a triple (used in a role).
Should it be in concepts?

(If only `rdf:subject` were available ...)